engine/orchestration/src/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -1577,8 +1577,9 @@ private void advanceStop(final VMInstanceVO vm, final boolean cleanUpEvenIfUnabl
         }
 
         try {
-            _networkMgr.release(profile, cleanUpEvenIfUnableToStop);
-            s_logger.debug("Successfully released network resources for the vm " + vm);
+            s_logger.debug("Not releasing network resources until expunge command is sent");
```

Comment:
I decided to remove try-block as it only logged info, and include it on method's javadoc, as I don't think it could be worth logging. Do you agree?